src/fpnv/fpnv.ts

```diff
@@ -0,0 +1,41 @@
+/*!
+ * @license
+ * Copyright 2017 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { App } from '../app';
+import { BaseFpnv } from './base-fpnv';
+
+/**
+ * Fpnv service bound to the provided app.
+ */
+export class Fpnv extends BaseFpnv {
```

Comment:
I'm not sure there's much value in splitting `Fpnv` from `BaseFpnv` ... it's a common Java idiom but I'm not sure it's that useful, especially in this case. You can't use FPNV without an app so I think it's fine to make just a single `Fpnv` class that requires an app